content/publication_resources/bibtex/lahti.bib

```diff
@@ -1,3 +1,19 @@
+@Article{Erawijantari2023,
+  author =	 {Pande Putu Erawijantari and Ece Kartal and José Liñares-Blanco and Teemu D. Laajala and 
+  Lily Elizabeth Feldman and The FINRISK Microbiome DREAM Challenge and ML4Microbiome Communities and 
```

Comment:
I think "The FINRISK.." should be put in {curly brackets} for correct rendering, same for ML4microbiome..